openshift/openshift-acme.template.yaml

```diff
@@ -120,6 +123,22 @@ objects:
             limits:
               cpu: ${CPU_LIMITS}
               memory: ${MEMORY_LIMITS}
+          livenessProbe:
+            exec:
+              command:
+              - /bin/sh
+              - -c
+              - "ps -A | grep openshift-acme"
```

Comment:
If all you care about is the existence of the process, I'd rather go with `pkill -0 openshift-acme` or `killall -0 openshift-acme`. If you have `ps` you should have `pkill`.

Reply:
pkill confirmed

Reply:
I care more than just the existence of the process, but there is nothing exposed in the controller that provides any liveness or readiness checks. This is my probem - are these checks even valuable?

Reply:
The natural route here would be to go upstream and get some kind of liveness/readiness added to the controller. 

Question is probably for @rrati: do we prioritize cleaning as many failures as possible, or doing them properly? Properly means going upstream and waiting for that cycle to complete, which may not match with deliverable timelines. Cleaning fast means adding these checks temporarily while we wait for upstream changes.

Reply:
The goal of this is to improve our services and to validate DVO's checks.  This is a valid check, so the solution should be correct as well.  Unless we can avoid it we should always do the fix properly.

We have the ability to skip specific checks if needed, but if that is done we should log a ticket for how to fix this properly.